From 5a559e130d3d070cb584a8d6f5210bed04a33439 Mon Sep 17 00:00:00 2001
From: "timothy@hatcher.name"
 <timothy@hatcher.name@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Wed, 1 Feb 2017 17:46:12 +0000
Subject: [PATCH] AXObjectCache is missing two functions at link time when
 !HAVE(ACCESSIBILITY) https://bugs.webkit.org/show_bug.cgi?id=167691

Reviewed by Chris Fleizach.

* accessibility/AXObjectCache.h:
(WebCore::AXObjectCache::recomputeDeferredIsIgnored): Added empty stub.
(WebCore::AXObjectCache::performDeferredIsIgnoredChange): Added empty stub.


git-svn-id: http://svn.webkit.org/repository/webkit/trunk@211488 268f45cc-cd09-0410-ab3c-d52691b4dbfc

diff --git a/Source/WebCore/accessibility/AXObjectCache.h b/Source/WebCore/accessibility/AXObjectCache.h
old mode 100644
new mode 100755
index 1a50fef2280..4f2bc6d9cb5
--- a/Source/WebCore/accessibility/AXObjectCache.h
+++ b/Source/WebCore/accessibility/AXObjectCache.h
@@ -490,6 +490,8 @@ inline void AXObjectCache::handleFocusedUIElementChanged(Node*, Node*) { }
 inline void AXObjectCache::handleScrollbarUpdate(ScrollView*) { }
 inline void AXObjectCache::handleAttributeChanged(const QualifiedName&, Element*) { }
 inline void AXObjectCache::recomputeIsIgnored(RenderObject*) { }
+inline void AXObjectCache::recomputeDeferredIsIgnored(RenderBlock&) { }
+inline void AXObjectCache::performDeferredIsIgnoredChange() { }
 inline void AXObjectCache::handleScrolledToAnchor(const Node*) { }
 inline void AXObjectCache::postTextStateChangeNotification(Node*, const AXTextStateChangeIntent&, const VisibleSelection&) { }
 inline void AXObjectCache::postTextStateChangeNotification(Node*, AXTextEditType, const String&, const VisiblePosition&) { }
-- 
2.13.0

